tox.ini

```diff
@@ -52,13 +57,3 @@ commands =
 commands =
     python -m pip install -U pip
     make upgrade
-
-[gh]
-# https://pypi.org/project/tox-gh/
-python =
-    3.7 = py37
-    3.8 = py38
-    3.9 = py39
-    3.10 = py310
-    3.11 = py311
-    pypy3 = pypy3
```

Comment:
As tox-gh is no longer needed for associating Python versions with Tox environment names, this bit of config is ripped out.